engine/rpcserver.go

```diff
@@ -1658,6 +1658,13 @@ func (s *RPCServer) GetHistoricCandles(_ context.Context, req *gctrpc.GetHistori
 			Volume: candles.Candles[i].Volume,
 		})
 	}
+
+	if req.Sync {
+		_, err = kline.StoreInDatabase(&candles)
```

Comment:
A second thing, I decided to have seed = true before seeding exchanges. What do you think about suggesting the user run dbseed when exchange is not found?
```
.\gctcli.exe gethistoriccandlesextended -e=poloniex -pair=USDT-BTC -asset=spot -sync=true -i=86400
2020/08/05 11:34:07 rpc error: code = Unknown desc = exchange not found
```
Not a requirement, but helps understand that its to do with database exchange missing, and not that the exchange doesn't exist

Reply:
Yeah I will add the output to let the users know